src/index.js

```diff
@@ -164,6 +164,14 @@ if ( !! window.wcCalypsoBridge.isEcommercePlanTrial ) {
 			scope: 'woocommerce-admin',
 		} );
 	}
+
+	if ( window?.wcCalypsoBridge?.square_connect_url ) {
```

Comment:
I think even with this, `GetPaidWithSquareFill` is still loaded into browser since it's using a direct import at the top of the file. We can further optimize by replacing it with a lazy import. What do you think?

Reply:
Unfortunately, I haven't found a way to lazy load fills yet. They do not get rendered when I lazy load it (we had the same issue with my previous PR)

Reply:
Oh, okay then! Thanks for looking